uber/utils.py

```diff
@@ -14,8 +14,6 @@ def __init__(self, page, *args, **kwargs):
         args = [self.quote(s) for s in args]
         kwargs = {k: self.quote(v) for k, v in kwargs.items()}
         cherrypy.HTTPRedirect.__init__(self, page.format(*args, **kwargs))
```

Comment:
So hey I think I figured out why it was there, it was part of the http_to_https.py file which relied on it to do the actual redirection of http->https.

i.e. stuff that should likely go away.
@EliAndrewC definitely would like your eyes on this bit.  so far, it didn't look like anything referenced it so looks OK to delete this and the other stuff in https://github.com/magfest/ubersystem/commit/94fe1053820e51e4647b2e6e340583bd321fb92d